pages/ksphere/kommander/1.2/release-notes/index.md

```diff
@@ -10,74 +9,54 @@ enterprise: false
-
-<!-- TBD when 1.2 goes GA:
-# Release Summary
+**NOTE:** You must be a registered user and logged on to the support portal to download this product. New customers must contact your sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>
 
+# Release Summary 
 Kommander provides a command center for all your cloud native management needs in public Information as a Service (IaaS), on-premises, and edge environments. Kommander provides a multi-tenant experience to create, secure, and configure Kubernetes clusters and cloud native workloads. Additionally, Kommander enables teams to unlock federated and cost management, across multiple clusters, whether they are a new Konvoy cluster or existing 3rd party/DIY distribution.
--->
 
 # Supported Versions
-
 | Kubernetes Support | Version |
 | ------------------ | ------- |
 | **Minimum**        | 1.15.0  |
 | **Maximum**        | 1.17.x  |
 | **Default**        | 1.17.3  |
 
-# Kommander 1.2.0 Beta 1 (September 14th, 2020)
-
-## Features/Improvements
-
-- Improved catalog handling
-- Improved Access Control for editing/deleting actions
+# New Features and Capabilities 
+## Air gapped environments
+Kommander now supports installing and running in an air gapped environment, either on premises or in the cloud. Air gapping is ideal for instances where high-security is a must or where Internet connectivity is either undesirable or unavailable. For more details on setting up Kommander in an air gapped environment, see [Install Kommander air gapped][install_airgapped]. 
 
-## Bug Fixes
-
-- Fixed an issue where logout wasn't deleting any cookies
-- Smaller UX Bugs and Improvements
-
-## Component Versions
-
-- Addon: `1.2.0-9`
-- Chart: `0.11.3`
-- auto-provisioning (yakcl): `0.4.5`
-- kommaner-federation (yakcl): `0.4.5`
-- kommander-licensing (yakcl): `0.4.5`
-- UI: `6.22.0`
+## Component versions
+- Addon: `1.2.0-19`
+- Chart: `0.11.15`
+- kommaner-federation (yakcl): `0.5.1`
+- kommander-licensing (yakcl): `0.5.1`
+- UI: `6.37.0`
 - kommander-karma: `0.3.10`
-- kubeaddons-catalog: `0.1.12`
+- kubeaddons-catalog: `0.1.14`
 - kommander-thanos: `0.1.15`
-- kubecost: `0.1.12`
+- kubecost: `0.1.15`
 - grafana: `4.6.3`
 
-# Kommander 1.2.0 Beta 0 (August 27th, 2020)
-
-## Features/Improvements
-
-- Changed routing from hash-based to history-based to allow redirects after login.
-
-## Bug Fixes
-
-- Smaller UX Bugs and Improvements
-
-## Component Versions
-
-- Addon: `1.2.0-5`
-- Chart: `0.10.3`
-- auto-provisioning (yakcl): `0.4.2`
-- kommaner-federation (yakcl): `0.4.2`
-- kommander-licensing (yakcl): `0.4.2`
-- UI: `6.22.0`
-- kommander-karma: `0.3.10`
-- kubeaddons-catalog: `0.1.12`
-- kommander-thanos: `0.1.15`
-- kubecost: `0.1.12`
-- grafana: `4.6.3`
\ No newline at end of file
+## Fixed and Improved Issues 
+* Improved Access Control for editing/deleting actions. 
```

Comment:
Do any of these have COPS ticket numbers or can we add links to additional information in the help?

Reply:
I could not find any by spot checking a few. I only found D2IQ tickets. The filter https://jira.d2iq.com/issues/?jql=project%20%3D%20%22Customer%20Operations%22%20%20%20AND%20(fixVersion%20%3D%20%22Kommander%201.2%22%20OR%20fixVersion%20%3D%20%22Kommander%201.2.0%20Beta%200%22%20OR%20fixVersion%20%3D%20%22Kommander%201.2.0%20Beta%201%22%20OR%20fixVersion%20%3D%20%22Kommander%201.2.0%20RC%201%22%20%20%20OR%20fixVersion%20%3D%20%22Kommander%201.2.0%20RC%202%22)%20 also did not result in anything related to these